src/compas_view2/views/view.py

```diff
@@ -19,23 +19,20 @@ class View(QtWidgets.QOpenGLWidget):
         The view configuration.
     """
 
-    FRONT = 1
-    RIGHT = 2
-    TOP = 3
-    PERSPECTIVE = 4
+    VIEW_ANGLES = {"FRONT": 1, "RIGHT": 2, "TOP": 3, "PERSPECTIVE": 4}
 
     def __init__(self, app, view_config):
         super().__init__()
         self.setFocusPolicy(QtCore.Qt.StrongFocus)
         self._opacity = 1.0
-        self._current = View.PERSPECTIVE
+        self._current = self.VIEW_ANGLES[view_config["view_port"]]
```

Comment:
my point earlier. perhaps the name in the config and the name in the code should correlate...